nuxeo-java-client/src/main/java/org/nuxeo/client/NuxeoClient.java

```diff
@@ -230,7 +231,7 @@ protected Response request(String url, UnaryOperator<Request.Builder> method) {
             Request request = method.apply(requestBuilder).build();
             return retrofit.callFactory().newCall(request).execute();
         } catch (IOException e) {
-            throw new NuxeoClientException(e);
+            throw new NuxeoClientException("Error during call on url=" + url, e);
```

Comment:
From the Javadoc of `okhttp.Call#execute()`:

> ```
>   * @throws IOException if the request could not be executed due to cancellation, a connectivity
>   * problem or timeout. Because networks can fail during an exchange, it is possible that the
>   * remote server accepted the request before the failure.
> ```

So I suggest to replace the message by _"The request xxx could not be executed due to cancellation, a connectivity problem or timeout"_. But maybe it has already been described by OK HTTP.